### PR TITLE
nestedScrollEnabled

### PR DESCRIPTION
**Pull request by PellegriniGuilherme**
I added the nestedScrollEnabled property to the ScrollView of the BarChart to enable nested scrolling. This is useful when the BarChart is used inside a horizontal ScrollView, as without this property, the chart's scrolling is compromised.

### CODE REVIEW

**Comment by Abhinandan-Kushwaha** [comment]
Hi @PellegriniGuilherme 
Thanks for this valuable contribution.

Please undo the unnecessary reordering of import statements in your PR and please don't push the yarn.lock file.
Once you make these changes in your PR, I can merge it.

File: src/BarChart/index.tsx [hunk 1]
```diff
@@ -1,12 +1,12 @@
-import React, {useCallback, useEffect, useMemo, useRef} from 'react';
-import {Animated, Easing, View} from 'react-native';
+import { BarChartPropsType, useBarChart } from 'gifted-charts-core';
+import React, { useCallback, useEffect, useMemo, useRef } from 'react';
+import { Animated, Easing, View } from 'react-native';
+import BarAndLineChartsWrapper from '../Components/BarAndLineChartsWrapper';
+import { Pointer } from '../Components/common/Pointer';
+import { StripAndLabel } from '../Components/common/StripAndLabel';
+import { screenWidth } from '../utils';
```
Comment (Abhinandan-Kushwaha) [inline src/BarChart/index.tsx:7]: Please undo the unnecessary reordering of import statements

File: src/Components/BarAndLineChartsWrapper/index.tsx [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
   BarAndLineChartsWrapperTypes,
+  chartTypes,
   useBarAndLineChartsWrapper,
+  yAxisSides,
 } from 'gifted-charts-core';
+import React, { Fragment, useEffect } from 'react';
+import { I18nManager, ScrollView, StyleSheet, View } from 'react-native';
+import { renderHorizSections } from './renderHorizSections';
+import RenderLineInBarChart from './renderLineInBarChart';
+import RenderVerticalLines from './renderVerticalLines';
```
Comment (Abhinandan-Kushwaha) [inline src/Components/BarAndLineChartsWrapper/index.tsx:11]: Please undo the unnecessary reordering of import statements

File: yarn.lock
Comment (Abhinandan-Kushwaha) [inline yarn.lock:1]: Please don't push the yarn.lock changes.

**Comment by PellegriniGuilherme** [comment]
All changes have been made. Thank you!

**Review by Abhinandan-Kushwaha — APPROVED** [review]

**Review by Abhinandan-Kushwaha — APPROVED** [review]

**Comment by vasylnahuliak** [comment]
Missing types

**Pull request merged**